install_config/monitoring/modules/grouping-alerts.adoc

```diff
@@ -0,0 +1,35 @@
+== Grouping alerts
+
+_FIXME get missing info and convert to procedure_
+
+Once alerts are firing against the Alertmanager, it must be configured to know how to logically group them.
+
+For this example a new route will be added to reflect alert routing of the "frontend" team.
+
+First, add new routes. Multiple routes may be added beneath the original route, typically to define the receiver for the notification. The following example uses a matcher to ensure that only alerts coming from the service `example-app` are used.
+
+  global:
```

Comment:
I'd make this a code block, but that might be part of the cleanup you were planning.

Reply:
Yes, we won't use this text in the future, so not fixing it.